File: fizzbuzz.py
import sys

def fizz_buzz(n):
    for i in range(1, n):
        if i % 15 == 0:
            print "FizzBuzz"
        elif i % 3 == 0:
            print "Fizz"
        elif i % 5 == 0:
            print "Buzz"
        else:
            print i

if len(sys.argv) > 1:
    fizz_buzz(int(sys.argv[1]))

File: FizzBuzz.scala
def fizzBuzz(n: Int){
  (1 to n) foreach {
    case x if x % 15 == 0 => println("FizzBuzz")
    case x if x %  3 == 0 => println("Fizz")
    case x if x %  5 == 0 => println("Buzz")
    case x => println(x)
  }
}

